samples/Mvc.RazorPagePartial/Pages/Index.cshtml

```diff
@@ -0,0 +1,13 @@
+@page "{handler?}"
+@model IndexModel
+
+<div id="ajax-partial">
```

Comment:
Is this effectively identical to how it works in MVC of yore?

Reply:
As far as I can tell, yes. There's obviously niceties around generating the link and form `Ajax.BeginForm` \ `Ajax.ActionLink`, but the remainder of it looks to be pretty consistent with Mvc 5.